Detections/ASimAuthentication/imAuthSigninsMultipleCountries.yaml

```diff
@@ -23,20 +23,24 @@ query: |
   let threshold = 2;
   imAuthentication
   | where TimeGenerated > timeframe
-  | where EventType=='Logon' and EventResult=='Success'
+  | where EventType == 'Logon' and EventResult == 'Success'
   | where isnotempty(SrcGeoCountry)
-  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), Vendors=make_set(EventVendor), Products=make_set(EventProduct)
-    , NumOfCountries = dcount(SrcGeoCountry)
-    by TargetUserId, TargetUsername, TargetUserType
+  | summarize
+      StartTime        = min(TimeGenerated)
+      , EndTime        = max(TimeGenerated)
+      , Vendors        = make_set(EventVendor)
+      , Products       = make_set(EventProduct)
+      , NumOfCountries = dcount(SrcGeoCountry)
+      , Countries      = make_set(SrcGeoCountry)
+      by TargetUserId, TargetUsername, TargetUserType
   | where NumOfCountries >= threshold
   | extend timestamp = StartTime, AccountCustomEntity = TargetUsername
-
 entityMappings:
   - entityType: Account
     fieldMappings:
       - identifier: FullName
```

Comment:
No longer supports the FullName. Please replace with Name and UPNSuffix

Reply:
Added both Name and UPNSuffix as Strongest identifiers according to the documentation.
https://learn.microsoft.com/en-us/azure/sentinel/entities-reference